cli_reference/get_started_cli.adoc

```diff
@@ -19,8 +19,8 @@ logging in to create your first project.
 
 ifdef::atomic-registry[]
 [NOTE]
-Some sections of this document refer to OpenShift. {product-title} is based on
-OpenShift technology.
+Some sections of this document refer to {product-title}. {product-title} is
+based on {product-title} technology.
```

Comment:
@ahardin-rh Since this note is for atomic-registry only, this actually should be saying that some sections refer to OpenShift, and that Atomic Registry is based on OpenShift technology. The note is here so that every little instance of "OpenShift" didn't need to be updated just for the Atomic Registry docs.